core/src/main/java/tech/pegasys/web3signer/core/service/http/handlers/signing/eth2/Eth2SignForIdentifierHandler.java

```diff
@@ -149,7 +149,7 @@ private void signWithSlashingProtection(
       final Eth2SigningRequestBody eth2SigningRequestBody,
       final Bytes signingRoot,
       final String signature) {
-    try {
+    try (final TimingContext ignored = slashingMetrics.getDatabaseTimer().startTimer()) {
```

Comment:
Would be better to put this in the DbSlashingProtection class so it's as close as possible to the time taken for the db queries.